htdocs/user/class/user.class.php

```diff
@@ -2322,7 +2322,8 @@ public function getNomUrl($withpictoimg = 0, $option = '', $infologin = 0, $noto
 		$label.= '<b>' . $langs->trans('Name') . ':</b> ' . $this->getFullName($langs, '');
 		if (! empty($this->login))
 			$label.= '<br><b>' . $langs->trans('Login') . ':</b> ' . $this->login;
-		$label.= '<br><b>' . $langs->trans("EMail").':</b> '.$this->email;
+		$label.= '<br><b>' . $langs->trans("Email").':</b> '.$this->email;
+		$label.= '<br><b>' . $langs->trans("Phone").':</b> '.$this->phone;
```

Comment:
@ptibogxiv perhaps you can add `if (! empty($this->phone))` because in every box where phone is not fetched, it will be empty

Reply:
@frederic34 good idea! fix done ;)